pommerman/configs.py

```diff
@@ -31,6 +31,7 @@ def ffa_competition_env():
         'num_items': constants.NUM_ITEMS,
         'max_steps': constants.MAX_STEPS,
         'render_fps': constants.RENDER_FPS,
+        'env': 0,
```

Comment:
i think we should be deliberate and say the full entry point, e.g. `pommerman.envs.v0:Pomme`. That will be more configurable as well as clearer.